### PR TITLE
csp-2274 config fetchers

### PR DESCRIPTION
**Pull request by amirbenun**

### CODE REVIEW

File: _meta/config/cloudbeat.common.yml.tmpl [hunk 1]
```diff
@@ -1,7 +1,12 @@
 cloudbeat:
   # Defines how often an event is sent to the output
   period: 30s
-  files: [
+  fetchers:
```
Comment (eyalkraft) [inline _meta/config/cloudbeat.common.yml.tmpl:4]: Nice!
The file-system fetcher config struct seem to match this already but it needs to be validated somehow (manual test/ the better solution will be a simple test on `parseConfigFetchers` with the default configuration and a validation of the created parsed fetchers and their config).

**Review by eyalkraft — APPROVED** [review]
Beautiful!
Ideally next time two separate PR's (rename+refactor first, then rebase the original one)

**Review by DaveSys911 — APPROVED** [review]
LGTM

**Review by eyalkraft — APPROVED** [review]
Nice!

**Pull request merged**